code/online_challenges/src/codeforces/round_615/div3_problem_B.cpp

```diff
@@ -0,0 +1,67 @@
+#include<bits/stdc++.h>
+using namespace std;
+void solve()
+{
+	int n;
+        cin >> n;
+	int x,y;
+	string ans = "";
+	ll max = 0;
+	vector<vector<int> > arr(1001);
+	for(int i=0; i<n;i++)
+	{
+		cin >> x >> y;
+		if(max < x)
+			max = x;
+		arr[x].push_back(y);
+	}
+	for(int i=0;i<max+1;i++)
+	{
+		sort(arr[i].begin(),arr[i].end());
```

Comment:
no need to use this loop `for(int i=0;i<max+1;i++)`  you can directly sort using this `sort(arr.begin(),arr.end());`

Reply:
Actually, I have used ``vector<vector<int> > arr(1001)`` so every array must be sorted .That's why, a loop is needed and each array is sorted using ``sort(arr[i].begin(),arr[i].end());``

Reply:
Yeah, this `vector<vector<int>> ` make sense now.